src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.tsx

```diff
@@ -201,11 +199,37 @@ const AdvancedSearchResultDetailsList: React.FC<IAdvancedSearchResultDetailsList
         }
     };
 
+    const getAvailableProperties = () => {
+        let properties: string[] = [];
+        twins.forEach((twin) => {
+            properties = Object.keys(twin).concat(properties);
+        });
+        properties.sort((a, b) => (a.toLowerCase() > b.toLowerCase() ? 1 : -1));
+        return new Set(properties);
+    };
+    const updateColumns = (columnToAdd: string) => {
+        setSelectedColumnNames((currentValue) => {
+            return currentValue.concat(columnToAdd);
+        });
+    };
+    const deleteColumn = (columnToRemoveKey: string) => {
+        setSelectedColumnNames((currentValue) =>
+            currentValue.filter((col) => col !== columnToRemoveKey)
+        );
+    };
     return (
         <section className={classNames.root}>
-            <h3 className={classNames.listHeader}>
-                {t('advancedSearch.results', { twinCount })}
-            </h3>
+            <div className={classNames.listHeaderAndIcon}>
+                <h3 className={classNames.listHeader}>
+                    {t('advancedSearch.results', { twinCount })}
+                </h3>
```

Comment:
it doens't default to the right list. I don't see the query columns getting added.
![image](https://user-images.githubusercontent.com/57726991/193125539-abd8f776-b270-4267-9548-5095ccbe918e.png)